modules/core/src/main/scala/org/tessellation/cli/method.scala

```diff
@@ -30,6 +30,10 @@ import fs2.io.file.Path
 
 object method {
 
+  val snapshotInterval: NonNegLong = 1000L
```

Comment:
You open a config in `core` module and you see `snapshotInterval` variable set to 1000L. Does it mean that each snapshot will be created with interval 1000? I mean - names are ambiguous

Reply:
Renamed the variables to `ordinalTrustUpdateInterval` and `ordinalTrustUpdateDelay`.